docs/pages/api-docs/data-grid.md

```diff
@@ -103,8 +103,8 @@ Api of the `components` props of type `GridSlotsComponent`
 | <span class="prop-name">Toolbar</span> | <span class="prop-type">React.ElementType</span> | <span class="prop-type">GridToolbar</span> | Toolbar component rendered above the grid column header bar.|
 | <span class="prop-name">PreferencesPanel</span> | <span class="prop-type">React.ElementType</span> | <span class="prop-type">PreferencesPanel</span> | PreferencesPanel component that renders the ColumnSelector or FilterPanel within a Panel component.|
 | <span class="prop-name">LoadingOverlay</span> | <span class="prop-type">React.ElementType</span> | <span class="prop-type">LoadingOverlay</span> | Loading overlay component rendered when the grid is in a loading state.|
+| <span class="prop-name">NoFilteredRowsOverlay </span> | <span class="prop-type">React.ElementType</span> | <span class="prop-type">NoFilteredRowsOverlay </span> | No filtered rows overlay component rendered when the grid has no results after filtering.|
```

Comment:
Actually, we didn't reach an agreement in https://github.com/mui-org/material-ui-x/pull/1445#discussion_r616683146. I chose this one because having "filtered" in the name makes it explicit that it's used in the filtering feature.